xwiki-rendering-transformations/xwiki-rendering-transformation-macro/src/main/java/org/xwiki/rendering/internal/macro/DefaultMacroManager.java

```diff
@@ -132,8 +133,8 @@
                 throw new MacroNotFoundException(String.format("No macro [%s] could be found.", macroId.toString()));
             }
         } catch (ComponentLookupException e) {
-            throw new MacroLookupException(String.format("Macro [%s] failed to be instantiated.", macroId.toString()),
-                e);
+            throw new MacroLookupException(String.format("Macro [%s] failed to be instantiated"
+                + ".", macroId.toString()), e);
```

Comment:
Why putting the "." on the second line ?

Reply:
k